src/pyfmi/fmi.pxd

```diff
@@ -116,7 +116,7 @@ cdef class FMUModelBase(ModelBase):
     cdef FMIL.jm_callbacks* callbacks_standard
 
     #Internal values
-    cdef public object __t
+    cdef public object _t
```

Comment:
Should we move this declaration to `ModelBase` since it is also added again on line 194? Feel free to ignore but looks like a possible clean-up